Release notes 0.334.0 (#555)
trinodb/trino-python-client: CHANGES.md

[README](https://github.com/trinodb/trino-python-client/blob/master/README.md)
and the [PyPI page](https://pypi.org/project/trino/).

## Release 0.334.0

* Allow authentication over insecure channel.
   ([#530](https://github.com/trinodb/trino-python-client/pull/530))
* Add support for `with` statement with `Cursor` class.
   ([#537](https://github.com/trinodb/trino-python-client/pull/537))
* Fix `KeyError` when accessing missing `error_location` in `TrinoQueryError`.
   ([#535](https://github.com/trinodb/trino-python-client/pull/535)).

> * Fix type annotation for `mutual_authentication` parameter in Kerberos/GSSAPI
>   authentication.
>   ([#542](https://github.com/trinodb/trino-python-client/pull/542))

Reviewer: ```suggestion
```

* Add support for `SET AUTHORIZATION` and `RESET AUTHORIZATION` statements.
  ([#551](https://github.com/trinodb/trino-python-client/pull/551))
* Use default TLS port when URI scheme is `https`.
  ([#549](https://github.com/trinodb/trino-python-client/pull/549))

> * Flush redirect URI during OAuth authentication.
>   ([#548](https://github.com/trinodb/trino-python-client/pull/548))

Reviewer: ```suggestion
* Fix redirect URI not getting printed during OAuth authentication when using terminals that buffer output.
  ([#548](https://github.com/trinodb/trino-python-client/pull/548))
```

> * Use passed TLS verification mode when retrieving spooled results.
>   ([#546](https://github.com/trinodb/trino-python-client/pull/546))

Reviewer: ```suggestion
* Use TLS verification mode set on the connection when retrieving spooled results.
  ([#546](https://github.com/trinodb/trino-python-client/pull/546))
```
